### PR TITLE
Adds feedback from PM hands on session

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2627](https://openedx.atlassian.net/browse/DOC-2627)

Addresses questions and suggestions raised during the interactive PI session with Ed Svcs.

### Reviewers
- [ ] Subject matter expert: @mmacfarlane
- [ ] Subject matter expert: @akuryla
- [ ] Doc team review (sanity check): @pdesjardins @catong @srpearce 

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Squash commits

File: en_us/shared/exercises_tools/completion.rst [hunk 1]
```diff
@@ -83,8 +83,8 @@ Add the Completion Tool to a Course
 After you have enabled the Completion tool in Studio, to add the Completion
 toggle control to a unit in a course, follow these steps.
 
-#. In the course outline in Studio, locate the unit to which you want to add the
-   Completion toggle control.
+#. In the course outline in Studio, locate the unit to which you want to add
+   the Completion toggle control.
```
Comment (pdesjardins) [inline en_us/shared/exercises_tools/completion.rst:87]: Comment not specific to your changes: why is "Completion" capitalized. I'm guessing that this is not a proper noun and that it should not be capitalized.

File: en_us/shared/exercises_tools/poll_question.rst [hunk 2]
```diff
@@ -45,9 +45,11 @@ Before you can add a poll to your course, you must enable the poll tool in
 Studio or OLX.
 
 To enable the Poll tool in Studio, you add the ``"poll"`` key to the **Advanced
```
Comment (pdesjardins) [inline en_us/shared/exercises_tools/poll_question.rst:47]: Inconsistent capitalization of "Poll" tool. Line 53 is not capitalized. Again, not specific to your PR.

File: en_us/shared/exercises_tools/survey.rst [hunk 3]
```diff
@@ -48,8 +48,9 @@ Before you can add a survey to your course, you must enable the Survey tool in
 Studio or OLX.
 
 To enable the Survey tool in Studio, you add the ``"survey"`` key to the
-**Advanced Module List** on the **Advanced Settings** page. For more
-information, see :ref:`Enable Additional Exercises and Tools`.
+**Advanced Module List** on the **Advanced Settings** page. (Be sure to
+include the quotation marks around the key value.) For more information, see
+:ref:`Enable Additional Exercises and Tools`.
 
 Alternatively, you can use OLX to enable the poll tool.
```
Comment (pdesjardins) [inline en_us/shared/exercises_tools/survey.rst:55]: I think this should read "enable the survey tool."

**Comment by pdesjardins** [comment]
Looks good to me. My only comments were not about your changes. :+1:

**Comment by mmacfarlane** [comment]
I'm not really familiar with the completion tool, so may not be the best subject matter expert.

**Comment by lamagnifica** [comment]
@mmacfarlane the completion tool (and poll, and others) is in here only because of a suggestion @akuryla made about warning people to be sure to include the quotation marks when you add a key to the advanced settings page. This change was mostly about other comments people had about the PI tool.

**Pull request merged**